react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable prefer-const */
+import React, { useState } from 'react';
+
+export default function Accordion({ inputData }) {
+  const [activeIndex, setActiveIndex] = useState(undefined);
+
+  return (
+    <>
+      {inputData.map((entry) => Bellow(entry, activeIndex, setActiveIndex))}
```

Comment:
Please use JSX to make this call. It will be much more "React-y" and will serve you well when the HTML gets more complex.